Makefile

```diff
@@ -231,6 +231,13 @@ uninstall:
 		rm -f $(MANDIR)/man5/$$(basename $${i}); \
 	done
 
+windows:
+	GOOS=windows $(GO) build -ldflags '$(LDFLAGS_PODMAN)' -o bin/windows$@ $(PROJECT)/cmd/podman
+
+darwin:
```

Comment:
Instead of listing out targets here, I'd rather use a pattern rule like kubernetes-incubator/cri-o@e5031fcf9af7d6c78d21e38ab9f82d2392ba05e8 (although see also kubernetes-incubator/cri-o#1654).  Then you'd use `podman.cross.darwin.amd64` (for example) to cross-compile for that target without having to hard-code targets for all possible `GOOS` / `GOARCH` pairs.

It would be also nice to move these lines up to the neighborhood of the native `podman` target around line 100.

Reply:
i moved it as you suggested ... but im going to keep this really simple right now.   I agree that your recommendation is superior.

Reply:
> Instead of listing out targets here, I'd rather use a pattern rule...

I've filed #1034 with this.